core/chains/evm/log/pool_unit_test.go

```diff
@@ -0,0 +1,343 @@
+package log_test
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	log "github.com/smartcontractkit/chainlink/core/chains/evm/log"
```

Comment:
AFAIK it should be sorted/grouped, so that all `github.com/smartcontractkit/*` imports form a dedicated section.

Reply:
Oh thanks for pointing!
I assumed the gofmt tool will take care of right grouping too.
Fixed now.